alea/submitters/slurm.py

```diff
@@ -39,11 +26,37 @@ def __init__(self, *args, **kwargs):
         self.slurm_configurations = kwargs.get("slurm_configurations", {})
         self.template_path = self.slurm_configurations.pop("template_path", None)
         self.combine_n_jobs = self.slurm_configurations.pop("combine_n_jobs", 1)
+
+        self._eval_partition()
+        # suggested default arguments for utilix.batchq.submit_job
+        # for XENONnT collaboration
+        BATCHQ_DEFAULT_ARGUMENTS = {
+            "hours": 1,  # in the unit of hours
+            "mem_per_cpu": 2000,  # in the unit of Mb
+            "container": "xenonnt-development.simg",
+            "partition": self.partition,
+            "qos": self.qos,
+            "cpus_per_task": 1,
+            "exclude_nodes": self.exclude_nodes,
+        }
+
         self.batchq_arguments = {**BATCHQ_DEFAULT_ARGUMENTS, **self.slurm_configurations}
         self._check_batchq_arguments()
         super().__init__(*args, **kwargs)
         self.log_dir = self.outputfolder
 
+    def _eval_partition(self):
+        # automatically set default partition based on hostname
+        hostname = socket.gethostname()
+        if "midway3" in hostname:
+            self.partition = "lgrandi"
+            self.qos = "lgrandi"
+            self.exclude_nodes = ""
+        else:
```

Comment:
If people run on other servers, not RCCm, they will be confused by this condition.